lib/features/authentication/login/page/login.dart

```diff
@@ -50,33 +47,33 @@ class __LoginPageViewState extends State<_LoginPageView> {
         ),
         //
         Padding(
-          padding: EdgeInsets.only(left: 25.w, right: 25.w),
+          padding: EdgeInsets.only(top: 77.h, left: 25.w, right: 25.w),
           child: SizedBox(
-            height: 1.sh,
-            width: 1.sw,
+            height: double.infinity.sh,
```

Comment:
revert back to my changes to simply use
            height: 1.sh,
            width: 1.sw,
            there is no need for doube.infinity if you are already using 1.sh etc
            as 1.sh will take the full screen height , for example 2.sh will be two times of the screen height